typeScript homework

nevensh opened the pull request:
hey Elias i fix the node-modules .
i put it in the gitignore.
thank you

EliasSalom reviewed (changes requested):
the node modules are still here!!!

nevensh commented:
hey Elias .
 i deleted the node modules and I got it back , but it didn't work .
so I used the last method and it's work
i deleted the directory and i build another one .

EliasSalom reviewed (approved):
LGTM
